tests/utils/utils.go

```diff
@@ -279,14 +282,13 @@ func CreateSendCrossChainMessageTransaction(
 	ctx context.Context,
 	source SubnetTestInfo,
 	input teleportermessenger.TeleporterMessageInput,
-	fundedKey *ecdsa.PrivateKey,
+	relayerKey *ecdsa.PrivateKey,
```

Comment:
Lets leave this as `fundedKey` or maybe `senderKey`, because the sender does not need to be a relayer. Same with `CreateRetryMessageExecutionTransaction` below